packages/node_modules/sample-browser-auth-implicit/test/wdio/specs/implicit-grant-authentication.js

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-undef */
+
+import testUsers from '@ciscospark/test-helper-test-users';
+import {assert} from 'chai';
+import setValue from '../lib/set-value';
+
+describe(`sample-browser-auth-implicit`, () => {
+  describe(`authenticate using implicit grant flow`, () => {
+    const browserSpock = browser.select(`browserSpock`);
+    let spock;
+
+    before(`create Spock`, () => testUsers.create({count: 1, config: {displayName: `Spock`}})
+      .then((users) => {
+        [spock] = users;
+      }));
+
+
+    step(`browse to sample app, and verify the user is not authenticated`, () => {
+      browserSpock.url(`/packages/node_modules/sample-browser-auth-implicit`);
+      assert.equal(browserSpock.getText(`#authentication-status`), `Not Authenticated`);
+    });
+
+    step(`initiate implicit grant authentication flow`, () => {
+      browserSpock.click(`[title="authenticate"]`);
+    });
+
+    step(`enter user login information`, () => {
+      // Enter Email
+      browserSpock.waitUntil(() => browser.select(`browserSpock`).getTitle() === `Sign In - Cisco Spark`);
```

Comment:
Since `browserSpock` is already defined, do you need to go through the `browser.select` steps every time?

Reply:
indeed